src/routes/users/postUsers.ts

```diff
@@ -0,0 +1,54 @@
+import express, { Request, Response, NextFunction } from 'express';
+import db from '../../knex';
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import { validate, setLocales, en } from 'robust-validator';
+
+const router = express.Router();
+const JWT_SECRET = process.env.JWT_SECRET || 'key';
+const saltRounds = 10;
+
+// Validation setup
+setLocales(en);
+
+const createUserValidation = {
+  email: 'required|email',
+  password: 'required|min:6|max:50',
+  full_name: 'required|min:3|max:100', 
+};
+
+router.post('/', async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
This is not a correct way. You should export the async function only. You don't need to use `router.post` here. You already use `router.post` in the `index.ts` file. In here, you just need to export the function. That's all.